config

```diff
@@ -3598,6 +3599,9 @@ CONFIG_INFINIBAND_ADDR_TRANS=y
 # CONFIG_INFINIBAND_OCRDMA is not set
 # CONFIG_INFINIBAND_VMWARE_PVRDMA is not set
 CONFIG_INFINIBAND_IPOIB=y
+# CONFIG_INFINIBAND_IPOIB_CM is not set
+CONFIG_INFINIBAND_IPOIB_DEBUG=y
```

Comment:
Wonder why this is getting set

Reply:
To upgrade a config we need to do `make olddefconfig`. These are the default values either that are new or not currently set in the config.